From a34c340c622c24ee263d7c2bd924d94ca3aa9f31 Mon Sep 17 00:00:00 2001
From: Sergii Piatakov <sergii.piatakov@globallogic.com>
Date: Fri, 20 Oct 2023 15:21:54 +0300
Subject: [PATCH 2/2] formatting: use tabs instead of spaces

Signed-off-by: Mikhail Shikalovskyi <mikhail232003@gmail.com>
---
 calculator.cpp | 9 +++++++--
 calculator.h   | 6 +++---
 2 files changed, 10 insertions(+), 5 deletions(-)

diff --git a/calculator.cpp b/calculator.cpp
index d93e35b..15cf169 100644
--- a/calculator.cpp
+++ b/calculator.cpp
@@ -2,10 +2,15 @@
 
 int Calculator::Add (double a, double b)
 {
-	return a + b + 0.5;
+
+
+
+	return a + b;
+
+
 }
 
 int Calculator::Sub (double a, double b)
 {
-    return Add (a, -b);
+	return Add (a, -b);
 }
diff --git a/calculator.h b/calculator.h
index 3740907..d59d596 100644
--- a/calculator.h
+++ b/calculator.h
@@ -3,9 +3,9 @@
 
 class Calculator
 {
-    public:
-        int Add (double, double);
-        int Sub (double, double);
+	public:
+		int Add (double, double);
+		int Sub (double, double);
 };
 
 #endif//CALCULATOR_H
-- 
2.40.1.windows.1

